[Bug][Agent] The connector of PostgreSQL is closed

### What happened

1.
![企业微信截图_16759971732809](https://user-images.githubusercontent.com/58519431/218031489-5c578ec0-d8b6-4baf-a5a8-6b719d1bb205.png)
2.
`Caused by: org.postgresql.util.PSQLException: ERROR: replication slot "debezium" already exists`

### What you expected to happen

Continuous collection of `PostgreSql`

### How to reproduce

Use `PostgreSQL` source

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/PostgreSQLReader.java
+++ b/inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/PostgreSQLReader.java
@@ -28,7 +28,6 @@ import org.apache.inlong.agent.conf.JobProfile;
 import org.apache.inlong.agent.constant.AgentConstants;
 import org.apache.inlong.agent.constant.PostgreSQLConstants;
 import org.apache.inlong.agent.message.DefaultMessage;
-import org.apache.inlong.agent.metrics.AgentMetricItem;
 import org.apache.inlong.agent.metrics.audit.AuditUtils;
 import org.apache.inlong.agent.plugin.Message;
 import org.apache.inlong.agent.plugin.sources.snapshot.PostgreSQLSnapshotBase;
@@ -73,7 +72,6 @@ public class PostgreSQLReader extends AbstractReader {
     private static final Gson GSON = new Gson();
     private static final Logger LOGGER = LoggerFactory.getLogger(PostgreSQLReader.class);
     private final AgentConfiguration agentConf = AgentConfiguration.getAgentConf();
-    protected AgentMetricItem readerMetric;
     private String userName;
     private String password;
     private String hostName;
@@ -196,6 +194,7 @@ public class PostgreSQLReader extends AbstractReader {
         props.setProperty("connector.class", PostgresConnector.class.getCanonicalName());
         props.setProperty("database.server.name", serverName);
         props.setProperty("plugin.name", pluginName);
+        props.setProperty("slot.name", "slot" + instanceId);
         props.setProperty("database.hostname", hostName);
         props.setProperty("database.port", port);
         props.setProperty("database.user", userName);

